extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -261,11 +274,13 @@ export class ConsentUI {
           dev().assertElement(this.placeholder_), null);
     }
     classList.add(consentUiClasses.loading);
-    toggle(dev().assertElement(this.placeholder_), true);
     toggle(dev().assertElement(this.ui_), false);
     this.win_.addEventListener('message', this.boundHandleIframeMessages_);
     insertAfterOrAtStart(this.parent_, dev().assertElement(this.ui_), null);
-    return this.iframeReady_.promise;
+
+    return this.baseInstance_.mutateElement(() => {
```

Comment:
I think this would be a bit more natural as:

```javascript
return Promise.all([
  this.iframeReady_.promise,
  this.baseInstance_.mutateElement(() => {
    toggle(dev().assertElement(this.placeholder_), true);
  }),
]);
```
Its not really a performance difference in this case, but seems to more clearly communicate that the two async things are independent.